packages/react-components/react-rating-preview/stories/Rating/RatingReadOnly.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { Rating } from '@fluentui/react-rating-preview';
+
+export const ReadOnly = () => {
+  return <Rating readOnly defaultValue={3} ratingLabel={3} ratingCountLabel={'1,160'} />;
```

Comment:
nit: I think in general we should recommend using `value` with `readOnly` instead of `defaultValue` (which implies the user can change it from the default, but they can't since it's read-only).
```suggestion
  return <Rating readOnly value={3} ratingLabel={3} ratingCountLabel={'1,160'} />;
```